bin/counterfeiter.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+WD=$(dirname $0)
+WD=$(cd $WD; pwd)
+ROOT=$(dirname $WD)
```

Comment:
FYI, you can use:
```DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"```
To figure out the directory of the script itself. You can calculate the path of istio.io from there.

Reply:
Thanks.. Updated